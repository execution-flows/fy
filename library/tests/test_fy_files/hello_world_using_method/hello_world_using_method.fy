flow HelloWorld:
    method greet using constant

    def -> None:
        self._greet()
