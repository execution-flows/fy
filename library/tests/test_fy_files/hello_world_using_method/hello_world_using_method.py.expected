from base.execution_flow_base import ExecutionFlowBase

from test_fy_files.hello_world_using_method import Greet_UsingConstant_MethodMixin


class HelloWorld_Flow(
    # Method Mixins
    Greet_UsingConstant_MethodMixin,

    # Base
    ExecutionFlowBase[None]
):
    def __call__(self) -> None:
        self._greet()
